Code/GraphMol/Atom.cpp

```diff
@@ -621,6 +621,39 @@ double Atom::getMass() const {
   }
 }
 
+bool Atom::hasValenceViolation() const {
+  auto any_of = [](auto &vec, const auto &pred) {
```

Comment:
It seems like this lambda isn't necessary and I feel like it makes the code harder to read - having the `std::any_of()` down at line 654 would be clearer.